numba/cuda/cudadecl.py

```diff
@@ -366,6 +367,28 @@ class Cuda_fp16_cmp(ConcreteTemplate):
     return Cuda_fp16_cmp
 
 
+def _genfp16_comparison_operator(l_key):
+    @register_global(l_key)
+    class Cuda_fp16_operator_cmp(AbstractTemplate):
+        key = l_key
+
+        def generic(self, args, kws):
+            assert not kws
+
+            if len(args) == 2 and \
+                    (args[0] == types.float16 or args[1] == types.float16):
+                if (args[0] == types.float16):
+                    convertible = self.context.can_convert(args[1], args[0])
+                else:
+                    convertible = self.context.can_convert(args[0], args[1])
+
+                #print(args, convertible)
```

Comment:
Please remove commented code.

```suggestion
                #print(args, convertible)
```